doc/api/deprecations.md

```diff
@@ -289,8 +289,8 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/21153
     description: End-of-Life.
   - version:
+    - v6.12.6
     - v4.8.6
-    - v6.12.0
```

Comment:
Is the version supposed to change here? Same v6.12.0 to v6.12.6 in a bunch of places in this file

Reply:
Nope, definitely not supposed to be there. Note sure what happened here.